inference-engine/src/low_precision_transformations/src/depth_to_space.cpp

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "low_precision_transformations/depth_to_space.hpp"
+
+#include <algorithm>
+#include <details/caseless.hpp>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "low_precision_transformations/common/ie_lpt_exception.hpp"
+#include "low_precision_transformations/network_helper.hpp"
+
+using namespace InferenceEngine;
+using namespace InferenceEngine::details;
+
+void DepthToSpaceTransformation::transform(TransformationContext& context, CNNLayer& layer) const {
+    if (!canBeTransformed(context, layer)) {
+        return;
+    }
+
+    if ((layer.insData.size() == 0) || layer.insData.size() > 2) {
```

Comment:
Why you need this check if you already have 
```
    if (parents.size() != 1) {
        return false;
    }
```
on line 45?

Reply:
Avoid possible Klockwork issue: check size before usage